swig/java/addLoadLibraryCommand.cmake

```diff
@@ -5,6 +5,7 @@ if (NOT LIBRARY_FILE)
 set(LIBRARY_FILE JNIhelics)
 else()
 get_filename_component(LIBRARY_FILE ${LIBRARY_FILE} NAME_WE)
+STRING(REGEX REPLACE "^lib" "" LIBRARY_FILE ${LIBRARY_FILE})
```

Comment:
This should be UNIX only, right? We will still want the `lib` prefix. Alternatively, we can just add this to the documentation and ask users to load the shared library before calling any of the HELICS functions.

Reply:
I don't really know how the loadLibrary function actually works.  I know some things like that actually search for prepended versions as well.  So I don't know.  It woudn't be that hard to make it only run on UNIX, but obviously if this is needed then on UNIX it finding the name without the prepended lib so it might need to do the same in windows,  someone needed to test it out really.